src/views/MainComponent.vue

```diff
@@ -65,4 +65,7 @@ html {
   width: 100%;
 }
 
+.width-220 {
+  width: calc(100% - 220px);
+}
```

Comment:
These changes are so that we'll have a single source where we define the side menu width, while the main content width (router view) is defined according to the side menu width - changing the side menu width should directly affect the main content
In addition, changed the name to be more informative


```suggestion
$side-menu-width: 220px;
$router-view-width: calc(100% - #{$side-menu-width});

.side-menu-column {
  max-width: $side-menu-width;
  min-width: $side-menu-width;
  width: $side-menu-width;
}

.router-view-column {
  max-width: $router-view-width;
  min-width: $router-view-width;
  width: $router-view-width;
}
```